jbpm-recommendation-smile-random-forest/src/test/java/org/jbpm/prediction/randomforest/Series1Test.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.prediction.randomforest;
+
+import org.junit.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class Series1Test extends RandomForestPredictionServiceProcessTest {
+
+    /**
+     * This test shows how after testing with a mixed set (3x false, 17x true) how prediction (and probability)
+     * evolve when you then keep sending as "false" value to the outcome.
+     */
+    @Test
+    public void test1() {
```

Comment:
Method name is meaningless: try something that can even replace the comment above to describe the test.

when_MixingFalseAndTrueSet_Expect_FalseOutcomeWithConfidenceOverFiftyPercent (or something similar in this way)